src/components/avatar/avatar.story.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, boolean, select } from '@storybook/addon-knobs';
+import withReadme from 'storybook-readme/with-readme';
+import Section from '../../../.storybook/decorators/section';
+import Avatar from './avatar';
+import Readme from './README.md';
+
+storiesOf('Avatar', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withReadme(Readme))
+  .add('Avatar', () => {
+    const size = select('size', ['s', 'm', 'l'], 'l');
+
+    const wrapperSize = do {
+      if (size === 's') '26px';
+      else if (size === 'm') '48px';
+      else '100px';
+    };
+
+    return (
+      <Section>
+        <div
+          style={{
+            width: wrapperSize,
+            height: wrapperSize,
+          }}
```

Comment:
Can we use a className over inline styles here?

Reply:
I mean yeah, but is for the `story` so I was feeling bad about having to define a style for this 🤣

Reply:
Just use react emotion?

Reply:
> I didn't take a look myself. Does it even need a wrapper? 💭

Yep, it's needed just for the storybook because if not takes the whole width space 😅

Reply:
All good and fixed in 7208e4d